FAQ.md

```diff
@@ -68,31 +67,52 @@ Q. I have just installed Emacs, ProofGeneral and a proof assistant.
 
 A. You need to enable Unicode Tokens by the menu item:
 
+<<<<<<< HEAD
```

Comment:
Hi @psteckler, your copy of the FAQ.md file currently contains 7 "<<<<<<<-delimited" regions. Could you look at this?